keystore/kms/aws/keystore.go

```diff
@@ -0,0 +1,96 @@
+package aws
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+
+	baseKMS "github.com/cossacklabs/acra/keystore/kms"
+)
+
+// Keystore implementation of AWS Keystore
+type Keystore struct {
```

Comment:
hm, maybe lets name it as Encryptor? Because it conflicts with our component KeyStore that has much more responsibilities and used for different things. This struct will be used to encrypt/decrypt keys... Or any other name except KeyStore. KMSStore, or KMSEncryptor, or else. Or main idea to introduce kms.Keystore, another component different from keystore.Keystore?